core/class/jMQTT.class.php

```diff
@@ -689,8 +689,12 @@ public function postRemove() {
     public static function health() {
         $return = array();
         foreach(self::getBrokers() as $broker) {
+            // if(!$broker->getIsEnable() || $broker->getMqttClientState() == jMQTTBase::MQTTCLIENT_POK || $broker->getMqttClientState() == jMQTTBase::MQTTCLIENT_NOK)
+                // continue;
```

Comment:
Continue only if broker eqpt is disabled

Reply:
Done